src/components/Person.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import './Person.scss';
+
+export function Person({ name, sex, age, isMarried, partnerName }) {
+  const partner = sex === 'm' ? `wife's` : `husband's`;
+
+  return (
+    <>
+      <h2 className="Person__name">
+        {'My name is '}
+        {name}
+      </h2>
+      <p className="Person__age">
+        {'I am '}
+        {age}
+      </p>
+      <p className="Person__partner">
+        {isMarried
+          ? `My ${partner} name is ${partnerName}`
+          : 'I am not married'}
+      </p>
+    </>
+  );
+}
+
+Person.defaultProps = {
```

Comment:
use PropTypes with defaultProps

Reply:
As i understand react needs either isRequired for a property or a default setting .

Reply:
need add expected types for props (all props). Some props can be required, but if not, they must have a default value